.github/workflows/ci.yml

```diff
@@ -9,9 +9,6 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node: [14, 16]
```

Comment:
Do you think it's safe to remove the matrix? We still support Node 14 so it'd be nice to have a CI to validate that we don't ship changes that break our libraries on Node 14.

Reply:
You're right! That looks like an issue, we should read that matrix and take two versions of Node into account for this CI. 

I'd be thankful if you pushed this change.